scripts/deploy/main.bicep

```diff
@@ -54,6 +54,7 @@ param deployCosmosDB bool = true
   'Volatile'
```

Comment:
Please update the corresponding ARM JSON file (main.json) to reflect your changes. The "Deploy to Azure" button only works with JSON ARM templates.

Command to use:  az bicep build --file main.bicep